### DISCUSSION TITLE
Add blur and contrast

### DISCUSSION THREAD

**Discussion by Andrews-Linx** [ShankarBUS/ModernFlyouts#791] [Ideas]
I think that I've said everything with the title, but yep, would be great that you add blur and contrast behind the widget, you know with all that of the fluent design in Windows 11, sorry for my bad english :D

**Comment by prakharb5** [comment]
We already have a feature request for acrylic (blur) #66 

Regarding the fluent design to match Windows 11, it is planned for the next release of the app.
